app/assets/javascripts/startpage/startPageServices.coffee

```diff
@@ -29,6 +29,14 @@ startPageServices.service("StartPageStateData", () ->
     currentState = ""
     selectedTopics = {}
     responsesForSelectedTopics = {}
+    topicQuestionsDone = {} # Ids of the topics for which the questions have all been answered
+    numTopics = 0
+    currentTopic = 0 # current topic ID, used for determining if a topic is done for progress
+    latestTopic = 0 # latest topic ID
```

Comment:
Latest topic is the latest topic page that the user has reached. It's used alongside questionsLeft_static to not borke the progress bar when the user is not on the latest page. It makes it display the progress as it would be on the latest page, instead of reloading variables for not the most current page and breaking progress.